otter/convergence/service.py

```diff
@@ -121,7 +126,8 @@ def execute_convergence(tenant_id, group_id, log,
     log.msg('execute-convergence',
             servers=servers, lb_nodes=lb_nodes, steps=steps, now=now,
             desired=desired_group_state, active=active)
-    yield _update_active(scaling_group, active)
+    if group_status != ScalingGroupStatus.DELETING:
```

Comment:
Currently, user facing APIs do not get DELETING groups to be backward-compatible. This was the reason condition was added otherwise `_update_active` raised `NoScalingGroupError`.